engine/src/flutter/impeller/tessellator/tessellator.cc

```diff
@@ -161,16 +322,17 @@ VertexBuffer Tessellator::GenerateLineStrip(const Path& path,
   };
 }
 
-void Tessellator::TessellateConvexInternal(const Path& path,
+void Tessellator::TessellateConvexInternal(const PathSource& path,
                                            std::vector<Point>& point_buffer,
                                            std::vector<uint16_t>& index_buffer,
                                            Scalar tolerance) {
   point_buffer.clear();
   index_buffer.clear();
 
-  GLESVertexWriter writer(point_buffer, index_buffer);
+  GLESPathVertexWriter writer(point_buffer, index_buffer);
 
-  path.WritePolyline(tolerance, writer);
+  // path.WritePolyline(tolerance, writer);
```

Comment:
nit: commented out code.

Reply:
Inconceiv...  Oh, I thought I had deleted all of those. :(